### DISCUSSION TITLE
Query public timeseries without JWT Token?

### DISCUSSION THREAD

**Discussion by klausgollmer** [thingsboard/thingsboard#11708] [Q&A]
Is it somehow possible for public devices to query current timeseries data via http GET without using jwt tokens?
This would be a useful simplification, especially for educational purposes. 

Simple http(s)://host:port/api/plugins/telemetry/{entityType}/{entityId}/values/timeseries?keys=key1,key2,key3
results in "Authentication failed"

**Comment by aricooperdavis** [comment]
You can now do this with API Keys: https://thingsboard.io/blog/thingsboard-4-3-alarm-rules-2-0-enhanced-calculated-fields-api-keys-and-more/

Edit: Still not really public, but better than JWT...
